### ISSUE TITLE
[CoE Starter Kit - Feature]: Copilot Components added into the Inventory

### ISSUE THREAD

**Issue by jimpreston29** [microsoft/coe-starter-kit#9423]
### Describe the solution you'd like

Can the CoE get the inventory of the Copilot components that are Knowledge Sources.

we will use this as part of our governance to check with knowledge sources are being used

this will be related to the Copilots

![Image](https://github.com/user-attachments/assets/80705661-b107-44d8-b8d6-e58beb6cf95c)

### Describe alternatives you've considered

I could try to build it myself but would like to know if can be done in the product

**Comment by davidwvansickle** [comment]
Yes, when importing the CoPilot videos fail, the msdn_product table doesn't have an entry for "CoPilot" as a category that's surfaced in the column for Power Platform Product when importing to Community Hub Content Table. We removed the entries and it imported it with blanks for the column.  Can this be added?

**Comment by pbattini** [comment]
@jimpreston29 Thanks for the enhancement request. We are adding this to our backlog.

**Comment by mohamrizwa** [comment]
This issue is being closed as its scope is now included in a newly created consolidated enhancement request for centralized management of orphaned components (Apps, Flows, Connection References) in the CoE Starter Kit.

New Issue Reference: [[CoE Starter Kit - Feature]: https://github.com//issues/10338

Thank you for your contributions

**Comment by davidwvansickle** [comment]
Thank you!
